sdk/ai/azopenaiassistants/version.go

```diff
@@ -7,5 +7,5 @@
 package azopenaiassistants
 
 const (
-	version = "v0.2.1"
+	version = "v0.2.2"
```

Comment:
```suggestion
	version = "v0.3.0"
```

We've been doing "breaking change means rev the minor version for betas". (also, we'll need to update the changelog)

Reply:
fixed :)